src/Qir/Runtime/stdlib/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "qir-stdlib"
+version = "0.1.0"
+authors = ["Microsoft"]
+edition = "2021"
```

Comment:
I don't know what this line does but intuitively feel that the line might be meant to contain 20**22** instead of 2021.

Reply:
Surprisingly, the value "2022" fails to parse, and I believe that is because 2021 is the most recent edition. I didn't know this, but the Rust tools manage this value and new projects are automatically created with `edition = "2021"`. You can read more about this in the documentation at https://doc.rust-lang.org/edition-guide/introduction.html.